drivers/timer/riscv_machine_timer.c

```diff
@@ -207,6 +225,8 @@ static int sys_clock_driver_init(const struct device *dev)
 {
 	ARG_UNUSED(dev);
 
+	set_divider(CONFIG_RISCV_MACHINE_TIMER_SYSTEM_CLOCK_DIVIDER);
```

Comment:
I’d like to see what other think of this line that are supporting riscv, @npitre just did some work here and might care

does every riscv timer support a divider?

Reply:
Hi @teburd , riscv-privilege spec only mentions mtime and mtimecmp registers, so they are mandatory. 
HW support for timer divider seems like a helpful feature, but it is vendor-specific